cmd/lncli/commands.go

```diff
@@ -2505,41 +2507,36 @@ func sendToRoute(ctx *cli.Context) error {
 		route = routes.Route
 	}
 
-	req := &lnrpc.SendToRouteRequest{
+	req := &routerrpc.SendToRouteRequest{
 		PaymentHash: rHash,
 		Route:       route,
 	}
 
 	return sendToRouteRequest(ctx, req)
 }
 
-func sendToRouteRequest(ctx *cli.Context, req *lnrpc.SendToRouteRequest) error {
-	client, cleanUp := getClient(ctx)
-	defer cleanUp()
+func sendToRouteRequest(ctx *cli.Context, req *routerrpc.SendToRouteRequest) error {
+	conn := getClientConn(ctx, false)
+	defer conn.Close()
+
+	client := routerrpc.NewRouterClient(conn)
 
-	paymentStream, err := client.SendToRoute(context.Background())
+	resp, err := client.SendToRoute(context.Background(), req)
 	if err != nil {
 		return err
 	}
 
-	if err := paymentStream.Send(req); err != nil {
-		return err
+	marshaller := jsonpb.Marshaler{
```

Comment:
is this different from `printRespJSON`?

Reply:
I don't remember why I did that. Replaced by `printRespJSON`. But this is a debug commit anyway.